doc/user/pbr.rst

```diff
@@ -107,6 +107,12 @@ end destination.
    Use this individual nexthop as the place to forward packets when the match
    commands have matched a packet.
 
+.. clicmd:: set vrf unchanged|NAME
+
+   If unchanged is set, the rule will use the vrf table the interface is in
```

Comment:
it should be worth mentioning that this is not working with vrf netns backend.

Reply:
Will add that to the docs.

Reply:
Added